libs/chain-signatures/contract/src/crypto_shared/types.rs

```diff
@@ -27,6 +36,14 @@ pub enum PublicKeyExtended {
         /// Serialized compressed Edwards-y point.
         near_public_key_compressed: near_sdk::PublicKey,
         /// Decompressed Edwards point used for curve arithmetic operations.
+        #[cfg_attr(
+            all(feature = "abi", not(target_arch = "wasm32")),
+            schemars(with = "[u8; 32]"),
+            borsh(schema(with_funcs(
+                declaration = "<[u8; 32] as ::borsh::BorshSchema>::declaration",
+                definitions = "<[u8; 32] as ::borsh::BorshSchema>::add_definitions_recursively"
+            ),))
+        )]
```

Comment:
This code is also repeated below. I think we could avoid it by changing the type here to SerializableEdwardsPoint. Not sure if that is possible without breaking something else.